client/src/components/ProductListContainer.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import styled from "@emotion/styled";
+import ProductListItem from "./ProductListItem";
+import compareSrc from "../assets/compare_icon.svg";
+
+function ProductListContainer({ list }) {
+  return (
+    <Container>
+      <h4>{list.creationDate}</h4>
+      <ProductListItem list={list} />
+      <Link to={`/list/${list.id}/compare`}>
+        <img src={compareSrc} alt="Compare Icon" />
+        <p>Compare</p>
+      </Link>
+    </Container>
+  );
+}
+
+export default ProductListContainer;
+ProductListContainer.propTypes = {
+  list: PropTypes.shape({
+    id: PropTypes.number,
+    products: PropTypes.array,
+    creationDate: PropTypes.string,
+  }),
+};
+
+const Container = styled.div`
+  width: 90%;
+  margin: 1rem;
+  background-color: #8c493a;
```

Comment:
You could add a CSS variable for this color if you like.

Reply:
That's true. I will change it 💯